modules/op-running-as-rootless-in-container.adoc

```diff
@@ -0,0 +1,45 @@
+[id='op-running-as-rootless-in-container_{context}']
+= Running as rootless in container
+
+You can run as rootless inside a container. To run as a rootless user inside the container, you need your container to do some setup to add the `subuid`` and `subgid` with the latest shadow-utils.
```

Comment:
```suggestion
You can run builds as a non-root user inside a container. To run as a non-root user inside the container, you need your container to do some set up to add the `subuid` and `subgid` with the latest shadow-utils.
```
- The phrase "you need your container to do some setup to add the `subuid` and `subgid` with the latest shadow-utils" is ambiguous. Are you saying you need a container for **you** to perform some set up steps? Or that you need **your container** to perform some set up steps on your behalf?

Reply:
- Clarify "add the `subuid` and `subgid` with the latest shadow-utils." Do you mean "by using shadow utils to add `subuid` and `subgid`? Or do you mean "add the `subuid` and `subgid` **along with** the latest shadow-utils."
- The term shadow-utils might be unfamiliar to our users. I don't see any references to it in the openshift-docs repo. Is there a clearer term?